cypress/integration/inventory/keyboard-shortcuts.spec.js

```diff
@@ -18,7 +17,7 @@ const issnValue = `ISSN test value ${getRandomPostfix()}`;
 const hotKeys = InventoryHotkeys.hotKeys;
 const instanceTitle = `Instance_Test_Title_${getRandomPostfix()}`;
 
-describe('ui-inventory: keyboard shortcut', () => {
+describe('ui-inventory: keyboard shortcut', { retries: 3 }, () => {
   beforeEach('navigate to inventory', () => {
     cy.createTempUser([
       permissions.inventoryAll.gui
```

Comment:
here is the comment: ```    // TODO: Need to wait for the loading of saving the edited information.Reason: the robot runs quickly and the test drops.```

Reply:
The reason actually is important: Need to wait for the loading of saving the edited information
please specify the reason